### ISSUE TITLE
AE2 Wireless Terminal randomly stops showing items

### ISSUE THREAD

**Issue by 1lloydbr** [FTBTeam/FTB-Modpack-Issues#5407]
### Modpack

FTB NeoTech

### Modpack version

1.3.0

### Has the pack been modified

No

### Describe the bug

I can put items and it goes to my network, also when I fill a recipe from EMI pulls items from the network, but I can't see anything from it, neither take items from it. It happened as soon as I crafted and tried using it, I thought I messed up something on the Wireless Access Point so I tried many different setups, and nothing worked. At some point it started working, I think I just left to test some stuff on other map and once I came back it was working. I used it normally and then it stopped working again(as shown below). I tried switching maps, quitting minecraft and oppening again, and it didn't work.

### Steps to reproduce

Since it occurred randomly I'm not sure, but I'll describe anyways:
1. Have a Working AE2 network
2. Craft a Wireless Crafting Terminal and a Wireless Access Point
3. Connect the Wireless Access Point to your ae2 network, and put the WCT at the top-right slot to link it
4. Open the Wireless Crafting Terminal

### Expected behaviour

Have access to the network's inventory, being able to take and put items.

### Screenshots

AE2 Crafting Terminal: 
![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/137570141/64c0152c-be79-43cd-a31d-6b2a630a585b)

Controller: (yes, I tried using a decent energy source, still the same)
![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/137570141/69f3c948-82ee-425b-a831-a9c3243ffe20)

Wireless Access Point and the Wireless Crafting Terminal linked to it: 
![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/137570141/b3e483b1-fb3a-4983-9336-f6651a2d5c18)

Wireless Crafting Terminal:
![image](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/137570141/2c6893ae-c3e7-4993-b51b-ac4ac18bc0f7)

**Comment by bo-jensen** [comment]
When I first crafted the Wireless Crafting Terminal, it was configured to only show Fluids.
To fix this;
1. Open the Wireless Crafting Terminal
2. Click the 4th button from the top (half ingot, half bucket icon)
3. Check all types you want to see (I just checked all of them)

Hopefully this helps you :-)

If this is the issue you have, pack maker might need to look at the default config for the terminal.
